libs/custom-paper-handler/src/driver/driver.ts

```diff
@@ -477,53 +443,95 @@ export class PaperHandlerDriver {
     await this.transferOutGeneric(ScanCommand, undefined);
     debug('STARTING SCAN');
     let scanStatus = OK_CONTINUE;
+    let dataBlockBytesReceived = 0;
+    let width = -1;
     const imageData: Uint8Array[] = [];
 
+    // Assumptions:
+    // 1. There's at least 1 data block
+    // 2. Each data block can fit within a single `transferInGeneric` call
     while (scanStatus === OK_CONTINUE) {
       const rawResponse = await this.transferInGeneric();
       assert(rawResponse?.data);
 
       const responseBuffer = rawResponse.data.buffer;
```

Comment:
I think we're just getting lucky here that the returned `ArrayBuffer` is _only_ the data we care about and not some larger shared buffer managed by `usb`. Accessing the underlying `ArrayBuffer` of a `DataView` should _always_ take into account the `byteOffset` and `byteLength` values:

```suggestion
      const responseBuffer = new Uint8Array(
        rawResponse.data.buffer,
        rawResponse.data.byteOffset,
        rawResponse.data.byteLength
      );
```

Reply:
Hmm sorry I know you flagged this in a previous PR but looks like I missed this incorrect usage.

Reply:
https://github.com/votingworks/vxsuite/pull/3756/commits/bb9464c00ad71cfccdf46d47bd4cab15e96a6d7e